packages/dmn-js-decision-table/src/features/decision-table-head/components/DecisionTableHead.js

```diff
@@ -42,6 +42,15 @@ export default class DecisionTableHead extends Component {
     this._changeSupport.offElementsChanged(root.id, this.onElementsChanged);
   }
 
+  getIndexColumnWidth() {
+    const root = this._sheet.getRoot(),
+          rowsCount = root.rows.length;
+
+    const width = 40 + Math.floor(Math.log10(rowsCount)) * 8;
```

Comment:
🧑‍🔬

Reply:
So the idea here is to give some defined amount of pixels per each digit of the index number right?

Reply:
Exactly.

Reply:
1 -> 40px
10 -> 48px
100 -> 56px and so forth

Reply:
I will add a comment with short explanation.

Reply:
Yeah makes sense probably, it took me a while to get what's going on.

Reply:
OK done.

Reply:
How do we know a digit is always 8px?

Reply:
It's a rough estimation based on double-digit and triple-digit numbers' width.

Reply:
Should we maybe programmatically calcualte the total size? I'm not sure if this is likely but this approach may create problems if we end up changing the CSS in the future.

Reply:
This may be an overkill too though.

Reply:
TBH I am afraid that could complicate the code + cause reflow. I haven't tried it, though. I am happy with this rough solution as there are not many cases to handle.

Reply:
Personally I'm ok with the current solution too.